conn_others.go

```diff
@@ -20,7 +20,7 @@ var _ osConn = &conn{}
 type conn struct{}
 
 // dial is the entry point for Dial.  dial always returns an error.
-func dial(family int) (*conn, error) {
+func dial(family int, config *SockOpts) (*conn, error) {
```

Comment:
This won't build yeah?  Should be `*Config`.  I'll see if I can set up OSX builds in Travis to check the "others" piece too.